### PR TITLE
added play to install texlive on all nodes

### PR DESCRIPTION
**Pull request by andriy-safe-ai**
Tested on dev cluster.

### CODE REVIEW

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Yes. Check the help desk.

**Comment by andriy-safe-ai** [comment]
Changed to install texlive on all nodes.

**Comment by steven-basart** [comment]
This is borderline a no from me.  I don't want people running lots of python stuff to compile plots on the login node.  Compute node is also kind of meh where we want to make sure they don't request a GPU node to make plots.

File: playbooks/roles/cais-all/tasks/ol-7.yml [hunk 1]
```diff
@@ -72,6 +72,17 @@
     name: safe_yum
   ignore_errors: true
 
+# Install texlive
```
Comment (steven-basart) [inline playbooks/roles/cais-all/tasks/ol-7.yml:75]: Not on all nodes. Only on compute and we want to make sure they request a CPU only for this.

**Comment by andriy-safe-ai** [comment]
@steven-basart 

I'm not sure how we can enforce cpu only for this other than emphasizing that they should only request cpus. 

I've push a change to only install `texlive` on our compute nodes. I'm also added `dvipng` which is also needed by the user.

**Comment by steven-basart** [comment]
`dvipng` this one can be installed by pip or conda I think?

**Pull request closed without merging**